src/en/about-juju.md

```diff
@@ -2,12 +2,11 @@ Title: What is Juju?
 
 # What is Juju?
 
-Juju is a state-of-the-art, open source, universal modelling tool for service
-oriented architectures and application oriented deployments. Juju allows you to
-deploy, configure, manage, maintain, and scale cloud applications quickly and
-efficiently on public clouds, as well as on physical servers, OpenStack, and
-containers. You can use Juju from the command line or through its beautiful
-GUI.
+Juju is a state-of-the-art, open source modelling tool for operating software
+in the cloud. Juju allows you to deploy, configure, manage, maintain, and
+scale cloud applications quickly and efficiently on public clouds, as well as
+on physical servers, OpenStack, and containers. You can use Juju from the
+command line or through its beautiful GUI.
 
```

Comment:
This was the peer-reviewed-by-dozens text from the shared document a few months back. I am okay for updating it, but I checked the other place it is used ( https://jujucharms.com/how-it-works) and it seems that has been updated too, and now says this:

'What is Juju?

Juju is an open source application modelling tool that allows you to deploy, configure, scale and operate cloud infrastructures quickly and efficiently on public clouds such as AWS, GCE, and Azure along with private ones such as MAAS, OpenStack, and VSphere.

Its store allows access to a wide range of best practice solutions which you can deploy with a single command. You can use Juju from the command line or through its powerful graphical representation of the model in the GUI.'